### ISSUE TITLE
Pink screen when building to iOS

### ISSUE THREAD

**Issue by AnaRhisT94** [richardstechnotes/UnityARKitDetection#3]
Hi Richard! 
Thanks for this repo.

After following carefully your steps, I was able to build for iOS. but I'm getting a pink screen instead of camera stream in the app. Any ideas why this happens and how to solve it?

**Comment by r-b-us** [comment]
Not really. Maybe the pink screen is actually a detection rectangle covering the entire screen. You could try changing the color of Assets/Materials/Rectangle.mat to check if this is the case.

You could also look at the debug info in the XCode console and check the line that starts "Got:". This is from the line 150 in Assets/Scripts/Detection.cs so you can see what the variables are that are being displayed.

It is possible that something has changed in the year since I did this so that more recent versions of things work differently.

**Comment by AnaRhisT94** [comment]
> Not really. Maybe the pink screen is actually a detection rectangle covering the entire screen. You could try changing the color of Assets/Materials/Rectangle.mat to check if this is the case.
> 
> You could also look at the debug info in the XCode console and check the line that starts "Got:". This is from the line 150 in Assets/Scripts/Detection.cs so you can see what the variables are that are being displayed.
> 
> It is possible that something has changed in the year since I did this so that more recent versions of things work differently.

Hi, thanks for the reply!
I see that it's a common problem in the latest version of Unity.
https://www.google.com/search?q=unity3d+getting+a+pink+screen+when+compiling+to+ios&oq=unity3d+getting+a+pink+screen+when+compiling+to+ios&aqs=chrome..69i57j33.9321j1j7&sourceid=chrome&ie=UTF-8

I'll try older version of Unity and also will try to fix this error anyways and update asap.

Thanks for your help!

**Comment by r-b-us** [comment]
Well let me know how you get on.

Actually, if you check ProjectSettings/ProjectVersion.txt, it indicates the 2017 version. Maybe you opened it with a newer version? Even if you don't save the project, I think that the version gets updated.

**Comment by AnaRhisT94** [comment]
@r-b-us 
Hi Richard,
it works great on the 2017 version.

I'll update you when I manage to solve with 2018 and/or 2019 version.

It works pretty descent, I'll try to upgrade it and debug it with AR-Remote, hopefully it works on the 2017's version. Did you use that for debugging?

Also, did you try to train a model from scratch and run it? If yes, how did it work?
